CHANGELOG.md

```diff
@@ -222,6 +222,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Changed version of `github.com/iver-wharf/wharf-core` from v1.2.0 to v1.3.0.
   (#125)
 
+- Changed docker-build scripts for easier windows building. (#126)
```

Comment:
nit: Could use the filename "Dockerfile" instead of "docker-build scripts" to be more precise. IMO I understand "Dockerfile" better

```suggestion
- Changed Dockerfile for easier windows building. (#126)
```